indico_hub/server.py

```diff
@@ -158,6 +164,10 @@ def get_stats(
         'language': language,
         'debug': debug,
     }
+    # adding optional info and url
+    for field in kwagrs:
+        machine_data[field] = kwagrs[field]
+    machine_data['url'] = inst_data['url']
     result = es.index(index='reg_data', id=uuid, body=machine_data)
     logger.warning('communicating with es...')
```

Comment:
why is this a warning? (fyi, usually this kind of debug code should not be committed at all, and for testing a simple `print()` usually does the job just fine - but nothing wrong with using the logger of course)

Reply:
solved